usaspending_api/reporting/v2/views/agencies/agency_code/overview.py

```diff
@@ -0,0 +1,127 @@
+            .annotate(
+                recent_publication_date=Subquery(
+                    SubmissionAttributes.objects.filter(*agency_filters).values("published_date")
+                ),
+                recent_publication_date_certified=Subquery(
+                    SubmissionAttributes.objects.filter(*agency_filters).values("certified_date")
+                ),
+                tas_obligations=Subquery(
+                    ReportingAgencyTas.objects.filter(
+                        fiscal_year=OuterRef("fiscal_year"),
+                        fiscal_period=OuterRef("fiscal_period"),
+                        toptier_code=OuterRef("toptier_code"),
+                    )
+                    .annotate(the_sum=Func(F("appropriation_obligated_amount"), function="SUM"))
+                    .values("the_sum"),
+                    output_field=DecimalField(max_digits=23, decimal_places=2),
+                ),
+                missing_tas_accounts=Subquery(
+                    ReportingAgencyMissingTas.objects.filter(
+                        fiscal_year=OuterRef("fiscal_year"),
+                        fiscal_period=OuterRef("fiscal_period"),
+                        toptier_code=OuterRef("toptier_code"),
+                    )
+                    .annotate(count=Func(F("tas_rendering_label"), function="COUNT"))
+                    .values("count"),
+                    output_field=IntegerField(),
+                ),
+            )
+            .values(
+                "fiscal_year",
+                "fiscal_period",
+                "total_dollars_obligated_gtas",
+                "total_budgetary_resources",
+                "total_diff_approp_ocpa_obligated_amounts",
+                "recent_publication_date",
+                "recent_publication_date_certified",
+                "tas_obligations",
+                "missing_tas_accounts",
+            )
+        )
+        return self.format_results(result_list)
+
+    def format_results(self, result_list):
+        results = []
+        for result in result_list:
+            results.append(
+                {
+                    "fiscal_year": result["fiscal_year"],
+                    "fiscal_period": result["fiscal_period"],
+                    "current_total_budget_authority_amount": result["total_budgetary_resources"],
+                    "recent_publication_date": result["recent_publication_date"],
+                    "recent_publication_date_certified": result["recent_publication_date_certified"] is not None,
+                    "tas_account_discrepancies_totals": {
+                        "gtas_obligation_total": result["total_dollars_obligated_gtas"],
+                        "tas_accounts_total": result["tas_obligations"],
+                        "missing_tas_accounts_count": result["missing_tas_accounts"],
+                    },
+                    "obligation_difference": result["total_diff_approp_ocpa_obligated_amounts"],
+                }
+            )
```

Comment:
Just a suggestion, use it or ignore

```suggestion
        results = [
            {
                "fiscal_year": result["fiscal_year"],
                "fiscal_period": result["fiscal_period"],
                "current_total_budget_authority_amount": result["total_budgetary_resources"],
                "recent_publication_date": result["recent_publication_date"],
                "recent_publication_date_certified": result["recent_publication_date_certified"] is not None,
                "tas_account_discrepancies_totals": {
                    "gtas_obligation_total": result["total_dollars_obligated_gtas"],
                    "tas_accounts_total": result["tas_obligations"],
                    "missing_tas_accounts_count": result["missing_tas_accounts"],
                },
                "obligation_difference": result["total_diff_approp_ocpa_obligated_amounts"],
            }
            for result in result_list
        ]
```